agent/websockets/shim.go

```diff
@@ -88,7 +88,9 @@ const (
           msgs.forEach(function(msg) {
             self.onmessage({ target: self, data: msg });
           });
-        }
+        } else {
+	  console.error("Received messages but there is no onmessage handler defined");
```

Comment:
This seems fine in general, but it is probably better if we note *what* received the message.

e.g. "The websocket shim received" instead of just "Received"